frontend/frontend-filters-links.php

```diff
@@ -533,6 +534,24 @@ protected function get_queried_term_id( $tax_query ) {
 			'fields' => 'ids',
 		);
 		$terms = get_terms( $args );
+
+		$filtered_terms_by_lang = array_filter(
+			$terms,
+			function ( $term ) use ( $lang ) {
+				$term_lang = $this->model->term->get_language( $term );
+
+				return $term_lang->slug === $lang;
```

Comment:
`get_language()` may return `false`.

```suggestion
				return ! empty( $term_lang ) && $term_lang->slug === $lang;
```